scripts/utils/multisigexecutor.ts

```diff
@@ -1,22 +1,21 @@
-const hre = require("hardhat");
-const ethers = hre.ethers;
-const inquirer = require("inquirer");
+import { Contract } from "@ethersproject/contracts";
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+import { ethers } from "hardhat";
+import inquirer from "inquirer";
+import multisigAbi from "./multisig.json";
 
-const multisigAbi = require('./multisig.json');
+export class MultisigExecutor {
+  private _deployer!: SignerWithAddress;
+  private _multisigWrapper!: Contract;
 
-class MultisigExecutor {
-  constructor(autoSign = true, setupGas = false) {
-    this._setupGas = setupGas;
-    this._autoSign = autoSign;
-  }
+  constructor(private _autoSign = true, private _setupGas = false) {}
 
-  async connect(multisigAddress) {
-    this._deployer = await ethers.getSigner();
+  async connect(multisigAddress: string) {
+    [this._deployer] = await ethers.getSigners();
```

Comment:
does this work? 🤯 nice one

Reply:
moreover `getSigner` isn't meant to be used like this, even though it works 😅